addons/website_crm/views/website_visitor_views.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo><data>
+    <!-- website visitor views -->
+    <record id="website_crm_website_visitor_view_form" model="ir.ui.view">
+        <field name="name">website.crm.website.visitor.form</field>
```

Comment:
Tututut ! website.visitor.view.form.inherit.website.crm . I let you check other occurrences.